### PR TITLE
master-qa-204304

### PR DESCRIPTION
**Pull request by suilin**
https://liferay.atlassian.net/browse/LPS-204304

### CODE REVIEW

**Comment by suilin** [comment]
The test passed on my PR:https://github.com/suilin/liferay-portal/pull/714#issuecomment-1867278705.

File: portal-web/test/functional/com/liferay/portalweb/macros/CrossClusterReplication.macro [hunk 1]
```diff
@@ -47,9 +47,16 @@ definition {
 
 		FormFields.enableCheckbox(fieldName = "ccrEnabled");
 
-		FormFields.editTextMultiline(
-			fieldName = "ccrLocalClusterConnectionConfigurations",
-			fieldValue = "localhost:8080,localcluster");
+		if (${portalCluster} == "true") {
+			FormFields.editTextMultiline(
+				fieldName = "ccrLocalClusterConnectionConfigurations",
+				fieldValue = "localhost:9080,localcluster");
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/macros/CrossClusterReplication.macro:53]: Define both `localhost:9080,localcluster` and `localhost:8080,localcluster` when portal is clustered

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/CrossClusterReplicationES7.testcase [hunk 2]
```diff
@@ -656,18 +636,17 @@ definition {
...
 
 		AssertTextEquals.assertPartialText(
-			locator1 = "Message#ERROR",
-			value1 = "Connection refused");
-
-		AssertElementNotPresent(
 			key_connectionId = "localcluster",
-			locator1 = "SearchAdmin#CLUSTER_NAME");
+			locator1 = "SearchAdmin#CLUSTER_NAME",
+			value1 = "LiferayElasticsearchClusterTwo");
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/CrossClusterReplicationES7.testcase:641]: Assert read/write labels for these connections if it's not tricky to do so

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/CrossClusterReplicationES7.testcase [hunk 3]
```diff
@@ -587,7 +588,18 @@ definition {
...
+		User.logoutPG(
+			password = "test",
+			userEmailAddress = "test@liferay.com");
+
+		User.loginPG(
+			nodePort = 9080,
+			password = "test",
+			userEmailAddress = "test@liferay.com");
+
+		CrossClusterReplication.setupCrossClusterReplication(portalCluster = "true");
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/CrossClusterReplicationES7.testcase:600]: CCR doesn't need to be setup twice. The configs for the 8080 node should carry over to 9080. That being said, `ccrLocalClusterConnectionConfigurations` should be configured with `localhost:8080,localcluster` and `localhost:9080,localcluster` when there is a portal cluster.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/CrossClusterReplicationES7.testcase [hunk 4]
```diff
@@ -603,46 +615,14 @@ definition {
 			locator1 = "SearchAdmin#CLUSTER_NAME",
 			value1 = "LiferayElasticsearchClusterTwo");
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/CrossClusterReplicationES7.testcase:616]: Assert the read/write labels if it's not tricky to do so

**Comment by suilin** [comment]
ci:reopen

**Pull request closed without merging**